applications/delete-application.php

```diff
@@ -14,8 +14,8 @@
     } else {
         echo "Could not delete application " . MESSAGES_APPLICATION_ID . PHP_EOL;
```

Comment:
This may not be the time to address this, but the application ID here and throughout the application section doesn't match the specs.

Reply:
Let's capture this as an issue for the future, agree that this is not the time to address this.

Reply:
https://github.com/Vonage/vonage-php-code-snippets/issues/53